jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -341,6 +341,8 @@ public boolean isTrustedRemoteAddr(String remoteAddr) {
   }
 
   public RequestContext getRequestContext(Request baseRequest) {
+    if (baseRequest == null)
```

Comment:
I suggest using the Google Style guide in new code and applying the formatter in seperate PRs.
This should be:
```java
if (baseRequest == null) {
  return null;
}
```
but fine as is if you plan to run the formatter in the other PR after pulling this change.